### ISSUE TITLE
Cannot find module 'serverless/lib/plugins/aws/package/lib/mergeIamTemplates'

### ISSUE THREAD

**Issue by j0k3r** [functionalone/serverless-iam-roles-per-function#1]
I just tried the plugin and got that error:

```
  Serverless Error ---------------------------------------
 
  Serverless plugin "serverless-iam-roles-per-function" initialization errored: Cannot find module 'serverless/lib/plugins/aws/package/lib/mergeIamTemplates'
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Forums:        forum.serverless.com
     Chat:          gitter.im/serverless/serverless
 
  Your Environment Information -----------------------------
     OS:                     linux
     Node Version:           8.4.0
     Serverless Version:     1.26.0
```

I have defined it in my serverless.yml:

```yml
plugins:
    - serverless-webpack
    - serverless-pseudo-parameters
    - serverless-iam-roles-per-function

# ...

functions:
    api:
        handler: lambda.apiHandler
        iamRoleStatements:
            -
                Effect: 'Allow'
                Action:
                    - logs:CreateLogGroup
                    - logs:CreateLogStream
                    - logs:PutLogEvents
                    - ec2:CreateNetworkInterface
                    - ec2:DescribeNetworkInterfaces
                    - ec2:DeleteNetworkInterface
                    - sns:Publish
                Resource: '*'
```

**Comment by glicht** [comment]
Hi,

Thanks for reporting this. The plugin has a dependency on the serverless framework. It expects it to be installed as a dev dependency. As a quick work around simply install the serverless framework:
```
npm install --save-dev serverless
```
I'll check what is the best option for removing this dependency, so you can use the plugin also when the serverless framework is installed globally. Will update the bug.

**Comment by glicht** [comment]
I've just pushed a fix for this issue and published version 0.1.2 to npmjs. You should be able to use the plugin now with the serverless framework installed globally.

**Comment by j0k3r** [comment]
Awesome, thanks for the quick fix!
